pkg/generator/generator.go

```diff
@@ -194,10 +194,35 @@ func Generate(node *v1alpha1.Node) error {
 			case v1alpha1.TypeAESKey:
 				// TODO placeholder
 				node.Value = []byte("temp-placeholder")
+
 			default:
 				return errors.WithStack(fmt.Errorf("Unexpected aliasConfig.Type: '%v', in %v", node.AliasConfig.Type, node.Path))
 			}
 		}
+	case v1alpha1.TypeTrustStore:
+		switch length := len(node.Path); length {
+		case 2:
+			value, err := GetKeystore(node.Path)
+			if err != nil {
+				return errors.WithStack(err)
+			}
+			node.Value = value
+		case 3:
```

Comment:
If the length of the `node.Path` is 3, you need to check the type of the third key. In this case `caCopy`.